Automattic/jetpack
# Make paypal blocks work in FSE & widget editor

Partly addresses [calypso#61319](https://github.com/Automattic/wp-calypso/issues/61319) - it addresses the issue with the widget editor and FSE but not the customizer

#### Changes proposed in this Pull Request:

Paypal blocks didn't work in the FSE & widget editors, you could add and edit them, but they wouldn't actually display on the front-end unless you saved and then totally reloaded the page.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






Happy path:
  1. Switch to a theme that supports widgets, e.g. Twenty Nineteen
  2. Go to Appearance > Widgets
  3. Add a new Pay with Paypal block, fill out the details and press Save
  4. Go to the front-end and notice that the block is displayed

Other places:
As above, but also check adding the block to
 * Posts (should continue to work)
 * FSE: Use Twenty TwentyTwo theme and go to Appearance->Edit whatsit (didn't work before, **should work now**)
 * Appearance -> Customizer -> Widgets (didn't work before, **still doesn't work now**)

The Customizer doesn't work on current production - it allows you to add a widget to the footer and save, but when you view the site, the widget isn't there. Reloading the page, or opening it in the Widget Editor gives a partially saved widget - no title, description or image ![image](https://user-images.githubusercontent.com/93301/157424387-d5eedaec-57ab-4f27-8267-74dee7116220.png). After applying this patch, the experience is exactly the same - **there isn't a regression**.
